lib/raven/configuration.rb

```diff
@@ -127,6 +127,12 @@ def initialize
       self.sanitize_fields = []
       self.sanitize_credit_cards = true
       self.environments = []
+
+      self.release = ENV['HEROKU_SLUG_COMMIT']
+
+      if self.release.nil? || self.release.empty?
+        self.release = `git rev-parse --short HEAD`.strip
```

Comment:
It's also the fact that git may not even be available on the system -- unless you're saying thats safe (not a rubyist myself)